src/pages/Dashboard/Dashboard.tsx

```diff
@@ -17,15 +19,10 @@ class DashboardPage extends React.Component<AllProps> {
           <h1>Hippocampal Volume</h1>
         </PageSection>
         <PageSection>
-          <div>
-            <h1 className="pf-u-mb-md">Welcome to ChRIS UI site's Dashboard</h1>
-              <Alert
-                aria-label="welcome wagon"
-                variant="info"
-                title="Welcome!"  >
-                Welcome to ChRIS UI site's Dashboard
-              </Alert>
-          </div>
+          {/* UnComment to render
```

Comment:
do we need this comment?

Reply:
Since the component will be moved from the Dashboard to somewhere else, I thought it will be easier to understand. I have removed the comment in the commit.